src/System.Xml.XmlSerializer/src/System/Xml/Serialization/CodeIdentifiers.cs

```diff
@@ -20,20 +20,19 @@ namespace System.Xml.Serialization
     /// </devdoc>
     internal class CodeIdentifiers
     {
-        private Hashtable _identifiers;
-        private Hashtable _reservedIdentifiers;
+        private Dictionary<string, object> _identifiers;
+        private Dictionary<string, object> _reservedIdentifiers;
         private ArrayList _list;
```

Comment:
It's the System.Collections.ArrayList. The shim types have been removed. I would need to update all ```ArrayList``` to ```List<>``` as well.